xformers/triton/k_dropout.py

```diff
@@ -18,66 +31,8 @@
     triton.Config({}, num_warps=16),
 ]
 
-
-@triton.jit
-def _get_4_bin_masks(seed_ptr, rand_offsets, p):
-    seed = tl.load(seed_ptr)
-    rand1, rand2, rand3, rand4 = tl.randint4x(seed, rand_offsets)
-
-    # binarize masks, save registers
-    # NOTE: We keep the random numbers as is there (integers over uint32),
-    # and convert the threshold instead, for speed
-
-    # The initial distribution is over 2**32 -1
-    # and our float threshold  is in between [0, 1]
-    # The full computation is: `start_point + full range * p`
-    threshold = (4294967296.0 * p).to(tl.int32)
-    rand_mask1 = rand1 > threshold
-    rand_mask2 = rand2 > threshold
-    rand_mask3 = rand3 > threshold
-    rand_mask4 = rand4 > threshold
-
-    return rand_mask1, rand_mask2, rand_mask3, rand_mask4
-
-
-@triton.jit
-def _random_prune_and_scale(x, rand_mask, p, p_scale):
-    zero = 0.0
-
-    # generate all the random numbers for the block at once, then reshape
-    keep = tl.reshape(rand_mask, x.shape)
-
-    # prune and normalize in one go
-    x = tl.where(keep, (x * p_scale).to(x.dtype), zero.to(x.dtype))
-    return x
-
-
-@triton.jit
-def tile_random_drop(
-    x_ptrs,
-    y_ptrs,
-    block_mask,
-    use_bias,
-    bias,
-    rand_mask,
-    p,
-    p_scale,
-    ACTIVATION,
-):
-    x = tl.load(x_ptrs, mask=block_mask, other=0.0)
-
-    # optionally apply a fused bias
-    if use_bias:
-        x += bias
-
-    # optional: fused activation (while the data is in shared memory)
-    if ACTIVATION:
-        x = ACTIVATION(x)
-
-    # randomly prune (and scale) the resulting buffer, possibly a no-op
-    output = _random_prune_and_scale(x, rand_mask, p, p_scale)
-
-    tl.store(y_ptrs, output, mask=block_mask)  # output
+MAX_INT32 = 2147483647
+MAX_UINT32 = 4294967295
```

Comment:
nit: maybe use this for readability?
```suggestion
MAX_INT32 = (1 << 31) - 1
MAX_UINT32 = (1 << 32) - 1
```